Add a LoRA tutorial to W&B Inference pages (#2023)
wandb/docs: inference/tutorials/creating-lora.mdx

{"messages": [{"role": "user", "content": "Can submarines swim?"}, {"role": "assistant", "content": "Well howdy there, partner! Now that's a rootin'-tootin' puzzler if I ever heard one!  \n\nSee, submarines don't \"swim\" like a river trout at a Saturday fish-fry. They ain't got no fins flappin' or gills puffin'. Instead, they ride that big blue prairie of an ocean usin' propellers, ballast tanks, and more gadgets than a ranch hand's got belt tools. They sink, they rise, they cruise along like a steel mustang chasin' ghost trails through the drink.  \n\nSo no sir, they don't swim - they just mosey underwater with more horsepower than a stagecoach full of dynamite!"}]}
```

## Post-training

This script trains a LoRA adapter on examples from the JSONL file and uploads it to W&B as an artifact for use with the W&B Inference API or Playground.

At a high level, this script does the following:

> 1. Logs in to W&B. The [Hugging Face Transformers integration](https://docs.wandb.ai/models/integrations/huggingface) of W&B Models will automatically record training progress and metrics.

Reviewer: ```suggestion
1. Logs in to W&B. The [Hugging Face Transformers integration](https://docs.wandb.ai/models/integrations/huggingface) of W&B Models automatically records training progress and metrics.
```
As part of our style guide, we try to avoid the future tense as much as possible.

2. Loads the base model ([OpenPipe/Qwen3-14B-Instruct](https://huggingface.co/OpenPipe/Qwen3-14B-Instruct)) from Hugging Face.

> 3. Configure the LoRA using hyperparameters such as rank and alpha defined as constants near the top of the file.

Reviewer: ```suggestion
3. Configures the LoRA using hyperparameters, such as rank and alpha defined as constants near the top of the file.
```

> 4. Load the examples from the file into a Dataset and then run [SFTTrainer](https://huggingface.co/docs/trl/en/sft_trainer). By default all examples will be used.

Reviewer: ```suggestion
4. Loads the examples from the file into a dataset and then runs [SFTTrainer](https://huggingface.co/docs/trl/en/sft_trainer). By default, the script uses all examples.
```

> 5. Save the LoRA and upload it to W&B as an Artifact so it can be used with W&B Inference.

Reviewer: ```suggestion
5. Saves the LoRA and uploads it to W&B as an [Artifact](/models/artifacts) so it can be used with W&B Inference.
```
